create_github_from_jira.py

```diff
@@ -39,8 +42,47 @@ def cli(jira_token, jira_username, github_token, repo, labels, issue_numbers):
         )
         data = resp.json()
 
+        if debug:
+            pprint.pprint(data)
+
+        renderedContent = []
+        lastLanguage = ''
+        for content in data['fields']['description']['content']:
+            # Keep track of the last language seen to annotate the next code block encountered.
+            if 'attrs' in content:
+                if 'language' in content['attrs']:
+                    lastLanguage = content['attrs']['language']
+
+            if content['type'] == 'codeBlock':
+                renderedContent.append("```" + lastLanguage + "\n" + content['content'][0]['text'] + "\n```")
+                lastLanguage = ''
+
+            elif content['type'] == 'paragraph':
+                elements = []
+                for element in content['content']:
+                    if element['type'] == 'text':
+                        text = element['text']
+
+                        # Resolve inline code or links
+                        if 'marks' in element:
+                            if 'type' in element['marks'][0]:
+                                if element['marks'][0]['type'] == 'code':
+                                    text = '`' + text + '`'
+                                if element['marks'][0]['type'] == 'link':
+                                    text = '[' + text + '](' + element['marks'][0]['attrs']['href'] + ')'
+
+                        elements.append(text)
+
+                renderedContent.append("".join(elements))
+
         title = data['fields']['summary']
-        description = header.replace("{{TICKET}}", issue_number) + "\n" + "\n\n".join(map(lambda c: c['content'][0]['text'], data['fields']['description']['content']))
+        description = "\n\n".join(renderedContent) + "\n" + footer.replace("{{TICKET}}", issue_number)
+
+        if debug or dry_run:
```

Comment:
why dry_run adds output? looks like is taking more than one responsibility here, right?, probably if you want to see the generated description you need to `--debug` if you want to generate feedback to the user about the ticket that was about to be created I would suggest to do that in the `if dry_run:` with some extra text, like 
```
We haven't created the jira ticket because --dry-run flag was detected, the ticket that would be created would be:

Title: {{title}}
Description:
{{description}}
```